Fixes misleading note about entity bucket in transactions (#29)
Tapac/exposed-wiki: DAO.md

    ...
}
```

> Note: If you're using `UUIDTables` and `UUIDEntity` you can setup IDs manually inside transaction like this:

Reviewer: Nit: It's not just when using `UUIDTable` and `UUIDEntity`, so we can reword this.
```suggestion
Note: You can set up IDs manually inside a transaction like this:
```
